### PR TITLE
[I18n] Fix brand names and Caching method label

### PR DESCRIPTION
**Pull request by pedro-mendonca**
Description of these fixes:
- [x] Correct naming for ~memached~ Memcached, ~fcgi~ FastCGI, ~cachify~ Cachify
- [x] Fix `db: Caching method` to `Caching method: db` format, allowing each language to deal with the `%s: %s` (eg. french `%s : %s`)

### CODE REVIEW

File: inc/class-cachify.php [hunk 1]
```diff
@@ -578,8 +578,12 @@ public static function add_dashboard_count( $items = array() ) {
 				),
 				admin_url( 'options-general.php' )
 			),
-			esc_attr( strtolower( $method ) ),
-			esc_html__( 'Caching method', 'cachify' ),
+			sprintf(
+				/* translators: 1: "Caching method label"; 2: Actual method. */
+				esc_html__( '%1$s: %2$s', 'cachify' ),
+				esc_html__( 'Caching method', 'cachify' ),
+				esc_attr( strtolower( $method ) ),
```
Comment (lloc) [inline inc/class-cachify.php]: Removing the comma would help passing the tests against older versions.

Comment (pedro-mendonca) [reply inc/class-cachify.php]: You are right, sorry :)

**Comment by stklcode** [comment]
Thanks for this PR!

I do agree with almost all changes.


For "memcached" vs. "Memcached" it seems the author's use it with mixed capitalization as well. Seems capitalized in headlines, but all lowercase in the text ... most of the time.

> **About Memcached**
> memcached is a high-performance, distributed memory object caching system, generic in nature, but originally intended for use in speeding up dynamic web applications by alleviating database load.
> [...]
>
> **What it Does**
> memcached allows you [...]
> [...]
> With memcached, you can see [...]
>
> **Origin**
> Memcached was originally [...]

https://memcached.org/about

Any source for the "correct" spelling?

File: inc/setup/cachify.memcached.nginx.php [hunk 2]
```diff
@@ -71,7 +71,7 @@
 		<?php
 		printf(
 			/* translators: code */
-			esc_html__( 'If you have errors please try to change %1$s to %2$s This forces IPv4 because some servers that allow IPv4 and IPv6 are configured to bind memcached to IPv4 only.', 'cachify' ),
+			esc_html__( 'If you have errors please try to change %1$s to %2$s This forces IPv4 because some servers that allow IPv4 and IPv6 are configured to bind Memcached to IPv4 only.', 'cachify' ),
```
Comment (lloc) [inline inc/setup/cachify.memcached.nginx.php]: I believe (judging from what I read on the webpage) the name is `memcached`. It is written as `Memcached` only in headlines.

**Review by lloc — CHANGES_REQUESTED** [review]

File: inc/setup/cachify.memcached.nginx.php (hunk 2 above)
Comment (pedro-mendonca) [reply inc/setup/cachify.memcached.nginx.php]: Done

**Comment by pedro-mendonca** [comment]
> Any source for the "correct" spelling?

@stklcode the source was this plugin code, as there is another string [nginx Memcached setup](https://github.com/pluginkollektiv/cachify/blob/develop/inc/setup/cachify.memcached.nginx.php#L12), so I've included the uppercase change for consistency.
Still, I've already reverted that and excluded Memcached from the changes.

**Comment by lloc** [comment]
The PR is fine for me.

**Review by stklcode — APPROVED** [review]

**Pull request merged**